### ISSUE TITLE
Improve accessibility support

### ISSUE THREAD

**Issue by gorhill** [gorhill/httpswitchboard#401]
In email:

> Hello. For years I used Firefox in conjunction with my screen reader JAWS For Windows. Amongst the add-ons were NoScript, RequestPolicy, Ghostery and Adblock Plus. Recently I became acquainted with Mac OS X and use it in conjunction with VoiceOver. I use Safari primarily but use Chromium development builds because Safari can not download video files from websites. I just found HTTP Switchboard days ago and really like it. However, there are MAJOR accessibility issues.
> 
> Firstly, the largest hinderance is the control panel when the toolbar button is activated. I hear numbers and domain names but have absolutely no idea what is what. I also hear element names [all, XHR, frame, audio/video et cetera] but do not know why. I hear buttons which can be activated but most are unlabelled and ones that are make no sense to me. Nothing has help tags which go a long way for us who can not see. Needless to say, there must be major improvements because I can not use it to its maximum potential like my sighted counter-parts. All those unlabelled buttons in the settings areas I do not activate because I do not know their functions.
> 
> I probably should have stated the obvious in the beginning but ALL controls and elements must be properly marked up. Realistic or not all websites should comply to the WCAG accessibility guidelines but I do not know of any guidelines for extensions. I know Y-ARIA standards are amazing when coded properly. Accessibility for screen readers should be a top priority for any developer regardless whether it is an app or a browser add-on. This one is by far the most imperative for Chromium and that is why I really want to have the necessary back-end accessibility foundation in place so I can use it just like you. If you had to use VoiceOver I guarantee you would be just as frustrated.
> Indispensable Accessibility Information For App Developers
> • splinesoft/SSAccessibility · GitHub
> • Developer Resources
